### DISCUSSION TITLE
Error:The user has to be connected to the related git user.

### DISCUSSION THREAD

**Discussion by Had3r** [vercel/vercel#6105] [Help] [answered]
Hi everyone,

my problem is that after pushing the changes, pipeline on GitLab failed. I got message: `"Team access required to deploy."`.
So I request access (actually I have existing account and I am also an owner of the team), but this time I am getting
`"Error:The user has to be connected to the related git user."`. I checked configured metadata and all looks correct. 

What's more, a few days ago deploy was performed correctly (and that was also the last deploy), from that moment on, I have not changed anything in the settings either, I have not logged out, no changes. Deploy on other projects is done correctly, and I'm using the same account. 

What else can I pay attention to to find the cause of the problem?

**Comment by Thamasky** [comment, marked as answer]
Hi, 

I had the same issue today.
I finally figured it out:

They changed the webhook url from api.zeit.co to api.vercel.com, but apparently it's broken.
Go to gitlab -> settings -> webhooks

Then edit the webhook that vercel created and use api.zeit.co in stead of api.vercel.com

Happy coding!

**Reply by Had3r** [reply]
Thanks for the quick reply.

In my GitLab settings, however, I see a webhook set to `https: //api.zeit.co/v1/integrations/gitlab/webhook....`

So it looks like they reversed this change because deploy is already being done for me. It is a pity that this change was not implemented sooner, because it took me a while to debug it :)

**Reply by Thamasky** [reply]
Yeah, it cost me a few hours as well :')

**Reply by Thebenjamist** [reply]
How does on remedy this on Bitbucket??
